src/CallMessage.ts

```diff
@@ -112,3 +97,74 @@ export class CallMessage extends EventEmitter {
     return this._voiceEventSid;
   }
 }
+
+/**
+ * Parse CallMessage options. Used when constructing a CallMessage from the
+ * native layer, or by the Call and CallInvite classes when sending a
+ * CallMessage.
+ *
+ * @param options the CallMessage details.
+ *
+ * @internal
+ */
+export function parseCallMessageOptions(options: CallMessage.Options) {
```

Comment:
This function seems to validate the message object. Maybe call this validateCallMessage?